src/commands/installCosmosDBExtension.ts

```diff
@@ -3,11 +3,12 @@
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-import { AzExtTreeItem, IActionContext } from '@microsoft/vscode-azext-utils';
+import { AzExtTreeItem, IActionContext, nonNullValue } from '@microsoft/vscode-azext-utils';
 import { CosmosDBTreeItem } from '../tree/CosmosDBTreeItem';
 import { installExtension } from '../utils/installExtension';
 
-export async function installCosmosDBExtension(context: IActionContext, treeItem: AzExtTreeItem): Promise<void> {
+export async function installCosmosDBExtension(context: IActionContext, node?: AzExtTreeItem): Promise<void> {
+    const treeItem = nonNullValue(node);
```

Comment:
Initially I wasn't super happy that we have to do this here, however, after some thinking I actually think it's an improvement. 

Now if this command is mistakenly called without a `node` argument, we'll throw immediately instead of some unknown behavior (and the error will be easier to comprehend).